### PR TITLE
Code Structure Alignments

### PR DESCRIPTION
**Pull request by emiluzelac**
Aligning the structure for better readability as per indentation [standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation). GH/WPORG username emiluzelac

### CODE REVIEW

**Comment by carolinan** [comment]
Hi Emil. I am still seeing a large number of spacing issues, see the travis build.

**Comment by carolinan** [comment]
Or actually because the file needs to be split anyway I don't want to ask you to spend any more time on it, the tabbing etc will change again.
https://github.com/WordPress/twentytwenty/issues/168

File: functions.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ function twentytwenty_menus() {
         $locations = array(
             'footer-menu'    => __( 'Footer Menu', 'twentytwenty' ),
             'main-menu'      => __( 'Main Menu', 'twentytwenty' ),
-      			'shortcuts-menu' => __( 'Shortcuts Menu', 'twentytwenty' ),
+            'shortcuts-menu' => __( 'Shortcuts Menu', 'twentytwenty' ),
             'social-menu'    => __( 'Social Menu', 'twentytwenty' ),
 		    );
```
Comment (ntwb) [inline functions.php]: The above ~6 lines are indented using _spaces_ not _tabs_.

**Comment by carolinan** [comment]
Closed this because it has been fixed already.

**Pull request closed without merging**